src/run.js

```diff
@@ -347,7 +346,15 @@ const minimalcss = async options => {
     }
   }
 
-  // All URLs have been opened, and we now have multiple DOM objects.
+  // All URLs have been opened, and we now have multiple DOM (cheerio) objects.
+  // But first check that every spotted stylesheet (by <link> tags)
+  // got downloaded.
+  const missingASTs = [...allHrefs].filter(url => !stylesheetAsts[url])
+  if (missingASTs.length) {
+    throw new Error(
+      `Found stylesheets that failed to download (${missingASTs})`
```

Comment:
Here, making this string, there's an opportunity to better explain what *might* have failed. Is it necessary do you think?